HW03/ekaterina_balabanovich/spec/balabanovich_hw03_task_1_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require_relative '../balabanovich_hw03_task_1'
+
+RSpec.describe 'Task1' do
+  describe '#error_extract' do
+    context 'when one error occurred' do
+      let(:one_error_logs) do
+        <<~LOGS
+          10.6.246.103 - - [23/Apr/2018:20:30:39 +0300] 'POST /test/2/messages HTTP/1.1' 200 48 0.0498
+          10.6.246.101 - - [23/Apr/2018:20:30:42 +0300] 'POST /test/2/run HTTP/1.1' 200 - 0.2277
+          2018-04-23 20:30:42: SSL ERROR, peer: 10.6.246.101, peer cert: , #<Puma::MiniSSL::SSL: System error: Undefined error: 0 - 0>
+          10.6.246.101 - - [23/Apr/2018:20:31:39 +0300] 'POST /test/2/messages HTTP/1.1' 200 48 0.0290
+        LOGS
+      end
+
+      it 'returns one line' do
+        expect(error_extract(one_error_logs).lines.count).to eq(1)
+      end
+
+      it 'includes "error" in returned line' do
+        expect(error_extract(one_error_logs).include?('error')).to be true
+      end
```

Comment:
Honestly, I don't understand why do you need to check this statement, I would just check that we return the correct line
```suggestion
      it '...' do
        expect(error_extract(one_error_logs)).to eq("2018-04-23 20:30:42: SSL ERROR, peer: 10.6.246.101, peer cert: , #<Puma::MiniSSL::SSL: System error: Undefined error: 0 - 0>\n")
      end
```